projects/gis/src/app/main/main.component.html

```diff
@@ -3,6 +3,16 @@
 
     <geoengine-layer-list-menu></geoengine-layer-list-menu>
 
+    <ng-container *ngIf="config.BRANDING.HOMEPAGE as homepage">
+        <div class="divider"></div>
+
+        <a [href]="homepage.URL" mat-flat-button color="primary" class="homepage-button">
+            <mat-icon>keyboard_return</mat-icon>
+            <span></span>
+            <img [src]="homepage.BUTTON_IMAGE_URL" [alt]="homepage.BUTTON_ALT_TEXT" [matTooltip]="homepage.BUTTON_TOOLTIP_TEXT" />
```

Comment:
In fact, you can use it as text only, but you have two types of font colors

![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/dcc081bf-06bc-4218-87b8-beca16a6f929)

Reply:
efa38afd738ffc393da016aa8adf32354cd71291